doc/nrf/libraries/modem/nrf_modem_lib.rst

```diff
@@ -62,6 +64,23 @@ The behavior of the functions in the OS abstraction layer is dependent on the |N
 This is relevant for functions such as :c:func:`nrf_modem_os_shm_tx_alloc`, which uses :ref:`Zephyr's Heap implementation <zephyr:heap_v2>` to dynamically allocate memory.
 In this case, the characteristics of the allocations made by these functions depend on the heap implementation by Zephyr.
 
+Modem trace module
+******************
+The modem trace module is implemented in :file:`nrf\\lib\\nrf_modem_lib\\nrf_modem_lib_trace.c`.
```

Comment:
please no Windows paths, use `/`

Reply:
This file seems to have windows style paths for all similar file references. I will leave this as is to stay consistent with the rest of the file. Perhaps this can be improved here and the rest of NCS documentation at a later stage . @divipillai  FYI.

Reply:
We will fix this as part of a separate PR.